torch/onnx/symbolic.py

```diff
@@ -359,6 +359,11 @@ def leaky_relu(g, input, negative_slope, inplace=False):
     return g.op("LeakyRelu", input, alpha_f=_scalar(negative_slope))
 
 
+def hardtanh(g, self, min_val, max_val, inplace=False):
+    # See Note [Export inplace]
```

Comment:
The commet is not necessary.

Reply:
my bad, I taught it had to do with the ``inplace`` argument, but I guess I am wrong...  
It has been taken care of!